Apps/WebClient/src/Server/Models/FeatureToggleConfiguration.cs

```diff
@@ -0,0 +1,126 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.WebClient.Server.Models
+{
+    /// <summary>
+    /// Configuration data to be used by Health Gateway web client.
+    /// </summary>
+    /// <param name="Homepage">Settings for the home page.</param>
+    /// <param name="WaitingQueue">Settings for the waiting queue.</param>
+    /// <param name="NotificationCentre">Settings for the notification centre.</param>
+    /// <param name="Timeline">Settings for the timeline.</param>
+    /// <param name="Datasets">Settings for the data sets.</param>
+    /// <param name="Covid19">Settings for covid19 features.</param>
+    /// <param name="Dependents">Settings for dependents features.</param>
+    public record FeatureToggleConfiguration(
+        Homepage Homepage,
+        WaitingQueue WaitingQueue,
+        NotificationCentre NotificationCentre,
+        Timeline Timeline,
+#pragma warning disable CA1819
+        Datasets[] Datasets,
+#pragma warning restore CA1819
+        Covid19 Covid19,
+        Dependents Dependents);
```

Comment:
Nit: I'd suggest appending something like "Settings" to the names of the records (but leaving the property names as they are).

e.g.
```
        HomepageSettings Homepage,
        WaitingQueueSettings WaitingQueue,
        NotificationCentreSettings NotificationCentre,
        TimelineSettings Timeline,
#pragma warning disable CA1819
        DatasetSettings[] Datasets,
#pragma warning restore CA1819
        Covid19Settings Covid19,
        DependentSettings Dependents
```

Also applies to "DependentDatasets" => "DependentDatasetSettings".